src/diff/children.js

```diff
@@ -197,6 +197,14 @@ export function diffChildren(
 
 						newParentVNode._nextDom = oldDom;
 					}
+				} else if (
+					oldDom &&
+					oldVNode._dom == oldDom &&
+					oldDom.parentNode != parentDom
+				) {
+					// The above condition is handle null placeholders. See test in placeholder.test.js:
```

Comment:
Nit: I think it should be `to handle` here.

Reply:
Oh true, thanks!